src/ci/resources/tes_application.conf

```diff
@@ -67,3 +67,9 @@ backend {
     }
   }
 }
+
+services {
+    MetadataService {
+        metadata-keys-to-sanitize-utf8mb4 = ["submittedFiles:workflow", "commandLine", "outputs:metadata_type_validation.validate_boolean.boolean_output"]
```

Comment:
What was special about `outputs:metadata_type_validation.validate_boolean.boolean_output` that caused it to be added here? Might want a comment explaining.